src/main/java/jenkins/branch/BranchBuildStrategy.java

```diff
@@ -130,12 +130,42 @@ public boolean isAutomaticBuild(@NonNull SCMSource source,
      * {@link SCMHead} has been detected as created / modified.
      * @since 2.1.3
      */
+    @Deprecated
+    @SuppressWarnings("deprecation")
+    @Restricted(ProtectedExternally.class)
+    public boolean isAutomaticBuild(@NonNull SCMSource source,
+                                             @NonNull SCMHead head,
+                                             @NonNull SCMRevision currRevision,
+                                             @CheckForNull SCMRevision prevRevision,
+                                             @NonNull TaskListener listener) {
+        throw new UnsupportedOperationException("Modern implementation accessed using legacy API method");
```

Comment:
I think you'll need to chain the call through, where `lastSeenRevision` is also the `prevRevision`

Reply:
nevermind... I may have forgotten that these are supposed to be protected methods and are the SPI not the API